easybuild/easyconfigs/s/Stacks/Stacks-1.40-foss-2016a.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'ConfigureMake'
+
+name = 'Stacks'
+version = '1.40'
+
+homepage = 'http://creskolab.uoregon.edu/stacks/'
+description = """Stacks is a software pipeline for building loci from short-read sequences, such as those generated on
+ the Illumina platform. Stacks was developed to work with restriction enzyme-based data, such as RAD-seq, for the purpose
+ of building genetic maps and conducting population genomics and phylogeography.
+"""
+
+dependencies = [
```

Comment:
@RvDijk please move this down beneath `sources` for consistency with other easyconfigs